### ISSUE TITLE
Support for Iray emissive material

### ISSUE THREAD

**Issue by Bercraft** [danielbui78/yaluxplug#1]
I noticed that in scenes with emissive material like lights they are not converted or rendered the scene remains black as no lights is in it.
So can you please add support for iray emissive material of something like Reality plugin where you were able to convert every mesh to light.
Tank you.

**Comment by danielbui78** [comment]
Yes, you are correct: light emission is not currently read from Daz or set in LuxCoreRender yet.  I will put it on my todo list.  In the mean-time, you can manually add light emissions to your .SCN file by adding the following lines:

```
scene.materials.<material_name>.emission = "1.0 1.0 1.0" # Set Color
scene.materials.<material_name>.emission.gain = 100000.0 1000000.0 1000000.0 # Set lamp gain
```

You will need to experiment with gain values.  Typical Iray scenes will probably require gain of 10,000 - 100,000.

![Screenshot (37)](https://user-images.githubusercontent.com/11391924/149673905-6d83f768-e6ad-4d8e-855d-9cad735b5607.png)

**Comment by Bercraft** [comment]
Thank you for the fast reply, if you need help testing features i 'm ready to help!
Your is a great great work man!
